Add 2022-12-27 agenda and minutes (#88)
solid/notifications-panel: meetings/2022-12-27.md

* AC: +1 simplest. Keep current and publish notification-channel-types.

### Update data models
URL: https://github.com/solid/notifications/pull/135

>ACTION: eP: create issue on considering request as requirement or preference, and can server overwrite desired channel properties or has to reject if can't satisfy

>ACTION: eP to create issue discussing need for `channel` defined in the vocab or just in JSON-LD context.

* SC: Can we merge in this call after resolving https://github.com/solid/notifications/pull/135#issuecomment-1365984108 ?
* SC: Group considered the options and decided to go with `startAt` and `endAt`.
* SC: Merging.

### Add Solid Notification JSON-LD context
URL: https://github.com/solid/vocab/pull/62

* SC: Pass through once above issues/PRs are resolved.

 ### Update sequence diagrams
URL: https://github.com/solid/notifications/pull/133

>ACTION: eP to update diagram if necessary as separate PR

>* SC: Just noting here that we can come back to 133 after 135.

### The `state` feature

>ACTION: Create issue for `state`. Mentioned it is currently designed for single resource and doesn't handle multiple topics.

* SC: Would anyone like to record that issue?

> * AC: There are ways to do multiple resources but possible messy. Inclined to say it is a feature at risk.

Reviewer: ```suggestion
* AC: There are ways to do multiple resources, but possibly messy. Inclined to say it is a feature at risk.
```

* SC: Should we do that for the current spec?

> * AC: It is an optional feature and don't have implementations so can't say it is officially at disk. Given the target for the spec pre 1.0 it is okay to be removed for 1.0.

Reviewer: ```suggestion
* AC: It is an optional feature and doesn't have implementations, so can't say it is officially at disk. Given its target was for the pre-1.0 spec, it is okay to be removed for 1.0.
```

* SC: +1 to leave as is. Revisit.

> * RG: If a client decides to watch multiple resources it needs to know that it needs the most up to date state for all those resources. What other thoughts/ideas you have for implementing and how would that reflect the spec.

Reviewer: ```suggestion
* RG: If a client decides to watch multiple resources, it needs to know that it needs the most up-to-date state for all those resources. What other thoughts/ideas do you have for implementing, and how would that reflect the spec?
```

> * AC: I'd probably grab ETag and hash into composite or something where there is a composite structure but I don't like that structure and not sure if the state variable is best way to represent how we coordinate multiple resources. we may end up with something with a different model.

Reviewer: ```suggestion
* AC: I'd probably grab ETag and hash into composite or something where there is a composite structure, but I don't like that structure and not sure if a state variable is the best way to represent how we coordinate multiple resources. We may end up with something with a different model.
```

> * RG: We could just have an array of resources with a tuple for ETag..

Reviewer: ```suggestion
* RG: We could just have an array of resources with a tuple for ETag.
```

* AC: That may be the better model.
* SC: Lets go through that in the issue.
* AC: *Leaves meeting*

### Rename Subscription Resource to Subscription Service?
URL: https://github.com/solid/notifications/issues/63

### Target terminology

>ACTION: SC: will update LDN subscription type to use `target` instead of `inbox` and `sender` instead of `webid`.

* SC : Using `sendTo` instead of `target` but yes, updated towards LDNChannel2023 spec.

>ACTION: eP: will update draft PR for Webhook to use `sender` instead of `webid`.

### Reconsider notify:source
URL: https://github.com/solid/notifications/issues/74

> * SC: Should be resolved from PR 135 but vocab/JSON-LD context needs an update.

Reviewer: ```suggestion
* SC: Should be resolved from [PR 135](https://github.com/solid/notifications/pull/135), but vocab/JSON-LD context needs an update.
```
